examples/weather_station/src/app_task.cpp

```diff
@@ -254,6 +277,47 @@ void AppTask::MeasurementsTimerHandler()
 	sAppTask.UpdateClusterState();
 }
 
+void AppTask::OnIdentifyStart(Identify *)
+{
+	uint16_t identifyTimeS;
+	EmberAfStatus status = emberAfReadAttribute(kIdentifyEndpointId, ZCL_IDENTIFY_CLUSTER_ID, ZCL_IDENTIFY_TIME_ATTRIBUTE_ID, CLUSTER_MASK_SERVER, reinterpret_cast<uint8_t *>(&identifyTimeS), sizeof(identifyTimeS), NULL);
+	if (status != EMBER_ZCL_STATUS_SUCCESS) {
+		LOG_ERR("Reading identify time failed %x", status);
+		return;
+	}
+	sIdentifyFullDurationTimeMs = identifyTimeS * 1000;
```

Comment:
Do we need to maintain that? Isn't `OnIdentifyStop` called when the timeout expires?

Reply:
It is, but this variable is used to update cluster state every second with time left to the identify end